src/applications/check-in/components/pages/validate/ValidateDisplay.unit.spec.jsx

```diff
@@ -1,18 +1,51 @@
+/* eslint-disable camelcase */
 import React from 'react';
 import { expect } from 'chai';
 import { render } from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import { I18nextProvider } from 'react-i18next';
 import sinon from 'sinon';
 import { axeCheck } from 'platform/forms-system/test/config/helpers';
+import i18n from '../../../utils/i18n/i18n';
 import ValidateDisplay from './ValidateDisplay';
 
 describe('check-in experience', () => {
   describe('shared components', () => {
     describe('ValidateDisplay', () => {
+      let store;
+      const middleware = [];
+      const mockStore = configureStore(middleware);
+      const initState = {
+        checkInData: {
+          context: {
+            token: '',
+          },
+        },
+        featureToggles: {
+          check_in_experience_lorota_security_updates_enabled: false,
+        },
+      };
+      beforeEach(() => {
+        store = mockStore(initState);
+      });
       it('passes axeCheck', () => {
-        axeCheck(<ValidateDisplay />);
+        axeCheck(
+          <Provider store={store}>
```

Comment:
Could we add a test component to provide this boilerplate? I18AwareProvider or some such?

Reply:
Will revist in next phase of translation